release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1617,6 +1617,32 @@ In {product-title} 4.16, Patternfly 4 and React Router 5 were deprecated. The de
 [id="ocp-4-17-removed-features_{context}"]
 === Removed features
 
+[id="ocp-4-17-tls-1-2-ciphers_{context}"]
+==== Removed support for TLS 1.2 custom profile ciphers
+
+{product-title} no longer supports the following Transport Layer Security (TLS) 1.2 cipher suites that formed part of the custom TLS profile:
```

Comment:
Clarification: the custom TLS profile is self-created, so these ciphers didn't "form" any part of the custom TLS profile.
```suggestion
{product-title} no longer supports the following Transport Layer Security (TLS) 1.2 cipher suites.  Including them as a part of a custom TLS profile for TLS v1.2 will not give you the results you expect:
```

Reply:
It seems like this change didn't make it in the final push.